docs/4.x/tests-github.md

```diff
@@ -0,0 +1,54 @@
+---
+category: Develop
+previous: tests-js-tracker
```

Comment:
because the article will be in a different category, we need to change the `previous` and `next` entries or just not define them